### ISSUE TITLE
no controls working at all and back don't do anything 

### ISSUE THREAD

**Issue by manaboy** [Waterdish/Shipwright-Android#32]
Hi late to the show, I tried this on my h96 Max V58 rock chip RK3588 android 12 box and no controls work, vol down, up back button or any buttons on the remote isn't working, when I hook up my Xbox one s controller via Bluetooth or even official Nintendo switch joycons via Bluetooth nothing works. These controllers work fine with all other popular android emulator, retro arch, yuzu, dolphin, Aether recast etc.

I tried to use anydesk remote streamer that lets you use your phone to control with native touch which works fine with problematic native android games such as final fantasy Ever crisis but even that didn't work

**Comment by manaboy** [comment]
Ok so I connected up the Xbox 360 wireless adapter via USB and connected my x360 wireless controller to that and was able to press the back button to bring up the settings menu, in there I am able to map the button of my Xbox one s Bluetooth controller, only issue is, there is no way to map the back button to the back button of the Xbox one s controller, also the switch joycon minus button don't work to bring up the settings menu either, seems the back button of Bluetooth joycon and Xbox one s controller are different??? But when mapping other button's etc it seems to detect and work fine with all other buttons, just the back button don't work

**Comment by manaboy** [comment]
I accidently posted this in the pc fork and was told to post it here instead.

Thanks

**Comment by manaboy** [comment]
I accidently posted this in the pc fork and was told to post it here instead. 

Hence the copy and paste job

Thanks

**Comment by Waterdish** [comment]
Yeah the app has some weird issues with detecting certain controllers. That was my main motivation for adding the onscreen back button as a way to open the menu, but since you're likely not on a touchscreen device, that's not very useful. I'll revisit the controller auto detection code to see what might need fixing.

**Comment by manaboy** [comment]
Thanks, I really appreciate it. I corn across someone testing the Mario 64 android port and they write trying it on an old MX box, so I assume any old box should be able to run this game. I assume the back button on the remote and the back button on the touchscreen target different addresses.

**Comment by erlo68** [comment]
Hi, is this still beeing worked at? I tried using a XBOX-360 and a 8BitDo SF30 Pro, but both seem to not recognise either stick or D-Pad making moving impossible. Other buttons work fine. Even in button mapping they won't be recognised.
